inc/fields/filefield.class.php

```diff
@@ -43,15 +43,11 @@ public function displayField($canEdit = true) {
       if ($canEdit) {
          $required = $this->isRequired() ? ' required' : '';
 
-         echo '<input type="hidden" class="form-control"
```

Comment:
Are you sure about this deletion ?

Reply:
Yes : I think that I forgot to remove this line when I removed the HTML file upload with the version provided by Html::file() . several things confirm that : 
- the name of this input is the same as the name of the input generated by Html::file()
- when GLPI uses Html::file() it does not creates an extra input tag outside of the method

Reply:
yes : this input is useless

v2.6.0 before using Html::file()
https://github.com/pluginsGLPI/formcreator/blob/2.6.0/inc/fields/filefield.class.php#L8

v2.6.1 after using Html::file()
https://github.com/pluginsGLPI/formcreator/blob/2.6.1/inc/fields/filefield.class.php#L8

the input has always an empty value.

moreover the input following it has the same name.